### PR TITLE
Paula branch

### PR DESCRIPTION
**Pull request by paulapbpb**
Meal Steps UI

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class App extends React.Component {
     }
 
     return (
-      <View style={styles.container}>
+      <View>
```
Comment (stevenbuchko) [inline App.tsx:36]: Why did you get rid of the styling here?

File: src/screens/MealList/MealListScreen.tsx [hunk 2]
```diff
@@ -1,15 +1,21 @@
+import { Text, Card, CardItem, View } from 'native-base';
```
Comment (stevenbuchko) [inline src/screens/MealList/MealListScreen.tsx:1]: Make sure to delete any imports you're not using

File: src/screens/MealList/mealCard/MealCard.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { Text, Card, CardItem, View } from 'native-base';
-import { Image } from 'react-native';
+import { Text, Card, CardItem } from 'native-base';
+import { Image, StyleSheet, View } from 'react-native';
```
Comment (stevenbuchko) [inline src/screens/MealList/mealCard/MealCard.tsx:3]: No need to import StyleSheet here. Next time we meet, we can set up a linter for you to catch these small things

File: App.tsx (hunk 1 above)
Comment (stevenbuchko) [inline App.tsx:36]: Why did you remove the styling here?

File: src/screens/MealList/mealCard/MealCard.tsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ class MealCard extends React.Component<Props, State> {
 
     render() {
         return (
+            <View style={styles.container}>
```
Comment (stevenbuchko) [inline src/screens/MealList/mealCard/MealCard.tsx:18]: Hmm you shouldn't have to have done anything with this file

File: src/screens/MealList/mealCard/styles.tsx [hunk 5]
```diff
@@ -1,7 +1,9 @@
 import { StyleSheet } from 'react-native';
 
 const styles: any = StyleSheet.create({
+
     cardWrapper: {
+        alignSelf: 'center',
```
Comment (stevenbuchko) [inline src/screens/MealList/mealCard/styles.tsx:6]: Same with this one. This shoulda been kept untouched

File: src/screens/MealList/mealSteps/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        width: '100%',
+        height: 220,
+    },
+    mealImage: {
+        height: 220,
+        width: '100%',
+        flex: 1,
+    },
+    recipeNameCard: {
+        position: 'absolute',
```
Comment (stevenbuchko) [inline src/screens/MealList/mealSteps/styles.tsx:22]: I think this is going to give us issues on different screen sizes. You should almost never need to use absolute positioning

File: src/screens/MealList/styles.tsx [hunk 7]
```diff
@@ -2,6 +2,13 @@ import { StyleSheet } from 'react-native';
 
 const styles: any = StyleSheet.create({
 
+
+    cardWrapper: {
```
Comment (stevenbuchko) [inline src/screens/MealList/styles.tsx:7]: Shouldn't have needed to add this

**Review by stevenbuchko — COMMENTED** [review]
Looks really good overall! I do have some comments, but I think a good half of them are because you put the component in the MealListScreen to test. This is also my first time doing a code review / comments so I have no idea what this will show up like to you. We'll talk through these the next time we meet

File: App.tsx (hunk 1 above)
Comment (paulapbpb) [reply App.tsx:36]: Having a container on the whole page wasn't letting me put the recipe picture as a banner that covered the whole screen

File: src/screens/MealList/mealCard/MealCard.tsx (hunk 4 above)
Comment (paulapbpb) [reply src/screens/MealList/mealCard/MealCard.tsx:18]: Ah ok, sorry about that. Obviously feel free to leave

**Review by stevenbuchko — APPROVED** [review]
